src/navigation/navigation.ts

```diff
@@ -0,0 +1,49 @@
+import {Routes} from "../constants";
+import Dashboard from "../modules/dashboard";
+import RegattaList from "../modules/regattaList";
+import SettingsButton from "../components/header/settingsButton";
+import Settings from "../modules/settings";
+import RegattaItem from "../modules/regattaItem";
+
+export interface screenItem {
+    name: string,
+    icon?: [string, string],
+    component: (arg0: { navigation: any, route: any }) => JSX.Element,
+    options?: {}
+}
+
+export const tabs: screenItem[] = [
+    {
+        name: 'Dashboard',
+        icon: ['ios-information-circle', 'ios-information-circle-outline'],
+        component: Dashboard,
+        options: {
+            headerRight: SettingsButton
+        }
+    },
+    {
+        name: Routes.regattaList,
+        component: RegattaList,
+        icon: ['ios-information-circle', 'ios-information-circle-outline'],
+        options: {
+            headerRight: SettingsButton
+        }
+    },
+]
+
+export const screens: screenItem[] = [
+    {
+        name: Routes.regattaItem,
+        component: RegattaItem,
+        options: {
+            headerRight: SettingsButton
+        }
+    },
+    {
+        name: Routes.settings,
+        component: Settings,
+        options: {
+            headerRight: SettingsButton
+        }
+    }
+]
```

Comment:
It's something that react-navigation tried to get away from. This object-based schemes decreases code readability. If you don't have a specific reason for that, it's always better to define it explicitly directly in the JSX